src/seedu/addressbook/Main.java

```diff
@@ -31,12 +31,13 @@
     private List<? extends ReadOnlyPerson> lastShownList = Collections.emptyList();
 
 
-    public static void main(String... launchArgs) {
+    public static void main(String... launchArgs) throws FileDeletedException {
```

Comment:
Two points:
  - It is bad practice to throw an Exception from `main`: your program will crash; try to catch and solve it at some level
  - At such a high level, `main` doesn't (and shouldn't) know anything about a File; it only cares about running with the arguments